packages/node/src/utils.ts

```diff
@@ -27,10 +27,11 @@ export function deepReadDirSync(targetDir: string): string[] {
       const itemAbsPath = path.join(currentDirAbsPath, itemName);
 
       if (fs.statSync(itemAbsPath).isDirectory()) {
-        return [...absPaths, ...deepReadCurrentDir(itemAbsPath)];
+        return absPaths.concat(deepReadCurrentDir(itemAbsPath));
```

Comment:
Depending on how we transpile this, it might already be transpiled to something like [].concat(abspaths, [deppread]), doesn't hurt to omit the empty array though

Reply:
We transpile to es6, so the spread operator is kept.